pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -630,6 +636,26 @@ def U(self, sto, T, lithiation=None):
             out.print_name = r"U_\mathrm{p}(c^\mathrm{surf}_\mathrm{s,p}, T)"
         return out
 
+    def H(self, sto):
+        """Dimensional hysteresis [V]"""
+        Domain = self.domain.capitalize()
+        # tol = pybamm.settings.tolerances["U__c_s"]
+        # sto = pybamm.maximum(pybamm.minimum(sto,1-tol),tol)
+        inputs = {f"{self.phase_prefactor}{Domain} particle stoichiometry": sto}
+        h_ref = pybamm.FunctionParameter(
+            f"{self.phase_prefactor}{Domain} electrode OCP hysteresis [V]", inputs
+        )
+        return h_ref
+
+    def Q(self, sto):
+        """Capacity change as a function of stoichiometry"""
+        c_max = self.c_max
+        epsilon_s_av = self.epsilon_s_av
+        V_electrode = self.main_param.A_cc * self.domain_param.L
+        Li_max = c_max * V_electrode * epsilon_s_av
+        Q_max = Li_max * self.main_param.F / 3600
+        return Q_max * sto
+
```

Comment:
@rtimms This is the function that I added to the parameters, so that it's valid for individual phases when the composite electrode model is used.